python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -543,7 +543,7 @@ def test_struct_explode(data):
 
 
 def test_tz_localize():
-    data = Series(date_range("2000-04-01", "2000-04-03", freq="H"))
+    data = Series(date_range("2000-04-01", "2000-04-03", freq="h"))
```

Comment:
Were these "always wrong"?

Reply:
Not "always wrong" I think these aliases are just deprecated in pandas 2.x